ci/integration.js

```diff
@@ -5,7 +5,7 @@ const fs = require("fs");
 const _ = require("lodash");
```

Comment:
I would break out the `originalNotebook` into its own file. It makes it easier to make changes to the notebook and typically these mocked objects are usually in their own file if not folder.

Reply:
Should this be in it's own PR focusing on the ci integration tests? I can do it in this one, but it's already a PR with a pretty wide scope.

Reply:
NB: the other changes you asked for are stuff I'm introducing so I have no issue including those fixes in this PR.

Reply:
Yes, it should go into its own PR. You can just marked this comment as resolved.